NetCord/Gateway/GatewayIntents.cs

```diff
@@ -1,55 +1,133 @@
-﻿namespace NetCord.Gateway;
+namespace NetCord.Gateway;
 
+/// <summary>
+/// Intents are used to configure which events are passed to the bot via the gateway connection. Intents marked as privileged must be enabled in the application developer portal before being passed here, otherwise the connection will terminate with close code 4014.
+/// </summary>
 [Flags]
 public enum GatewayIntents : uint
 {
+    /// <summary>
+    /// Associated with the following events:<br/>
+    /// • Guild Events: <see cref="GatewayClient.GuildCreate"/>, <see cref="GatewayClient.GuildUpdate"/>, <see cref="GatewayClient.GuildDelete"/><br/>
+    /// • Role Events: <see cref="GatewayClient.RoleCreate"/>, <see cref="GatewayClient.RoleUpdate"/>, <see cref="GatewayClient.RoleDelete"/><br/>
+    /// • Channel Events: <see cref="GatewayClient.GuildChannelCreate"/>, <see cref="GatewayClient.GuildChannelUpdate"/>, <see cref="GatewayClient.GuildChannelDelete"/>, <see cref="GatewayClient.ChannelPinsUpdate"/><br/>
+    /// • Thread Events: <see cref="GatewayClient.GuildThreadCreate"/>, <see cref="GatewayClient.GuildThreadUpdate"/>, <see cref="GatewayClient.GuildThreadDelete"/>, <see cref="GatewayClient.GuildThreadListSync"/><br/>
+    /// • Thread User Events: <see cref="GatewayClient.GuildThreadUserUpdate"/>, <see cref="GatewayClient.GuildThreadUsersUpdate"/><br/>
+    /// • Stage Events: <see cref="GatewayClient.StageInstanceCreate"/>, <see cref="GatewayClient.StageInstanceUpdate"/>, <see cref="GatewayClient.StageInstanceDelete"/>
```

Comment:
Could you use for example that for formatting? https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#list

Reply:
Forgot that the doc comments supported that, will use

Reply:
Rewrote it with the proper tags